.gitignore

```diff
@@ -124,6 +124,7 @@ examples/plotting/20110520100000_nex_3d.nc
 examples/plotting/20110520100000_nexrad_grid.nc
 examples/plotting/narr-a_221_20110520_0000_000.nc
 examples/plotting/nsaxsaprppiC1.a1.20140201.184802.nc
+examples/plotting/034142.mdv  
```

Comment:
This line has trailing whitespace which for some reason means the file is not ignored.  Can you remove it?